site/content/en/docs/handbook/addons/gcp-auth.md

```diff
@@ -102,6 +102,10 @@ To resolve this problem, run:
 
 ## Adding new namespaces
 
+### minikube v1.29.0+
```

Comment:
can we make it tabbed? if it makes sense

Reply:
The text under is only a line or two, so I don't think tabbing makes sense here